### PR TITLE
maestro: Test a pod without a k8s svc accessing a service via TrafficTarget

### PR DESCRIPTION
**Pull request by draychev**
This PR adds a new pod (`flaneur`) to the bookbuyer + bookstore + bookwarehouse CI system. This pod does not have a related Kubernetes service.  The goal is to prove that even without a service, OSM will allow the pod to buy books from the bookstore. (Relies on an SMI policy with a flaneur service account)

This is a continuation of the work done with https://github.com/openservicemesh/osm/pull/1932

Proves that https://github.com/openservicemesh/osm/issues/1180 is fixed

### CODE REVIEW

File: ci/cmd/maestro.go [hunk 1]
```diff
@@ -22,13 +22,15 @@ const (
 	bookstoreV1Label   = "bookstore-v1"
 	bookstoreV2Label   = "bookstore-v2"
 	bookWarehouseLabel = "bookwarehouse"
+	flaneurLabel       = "flaneur"
```
Comment (eduser25) [inline ci/cmd/maestro.go:25]: "Flâneur (/flæˈnjʊər/; French: [flɑˈnœʁ]) is a French noun referring to a person, literally meaning 'stroller', 'lounger', 'saunterer', or 'loafer', but with some nuanced additional meanings (including as a loanword into English). Flânerie is the act of strolling, with all of its accompanying associations."

what

**Comment by draychev** [comment]
ref https://github.com/openservicemesh/osm/issues/2064

**Comment by ksubrmnn** [comment]
Are we still going with this approach? I believe @michelleN changes will be doing away with our ServiceAccount->Service conversions

**Comment by draychev** [comment]
@shashankram 

> 
> 
> Are we still going with this approach? I believe @michelleN changes will be doing away with our ServiceAccount->Service conversions

@ksubrmnn This PR adds a new test scenario; @shashankram meanwhile is swapping the mechanism my which we achieve this functionality.  It'd be good to have this demo/test - so we can easily validate the Service-less pod can access a mesh service.

**Comment by shashankram** [comment]
We already test this scenario: https://github.com/openservicemesh/osm/blob/main/tests/e2e/e2e_pod_client_server_test.go#L28, why do we need to test this via maestro again?

**Pull request closed without merging**